cg/src/domains/subjects/subjects.controller.js

```diff
@@ -16,10 +16,7 @@ class SubjectsController {
   }
 
   async updateConsent(req, res) {
-    await this.subjectsService.updateConsent(
-      req.subject.id,
-      req.body.processors || []
-    );
+    await this.subjectsService.updateConsent(req.subject.id, req.body.processors || []);
     res.json({ success: true });
```

Comment:
is this a standard to send `success` or could we just use HTTP code 204 to notify there is nothing to send but the request was successful?